website/src/shared/blocks/imageBlock.tsx

```diff
@@ -52,7 +53,11 @@ export function ImageBlock(props: ImageBlockProps) {
       />
       {(props.author || props.caption) && (
         <p className={css(ImageBlockCaptionStyle)}>
-          {props.caption} {props.author ? <>(Foto: {props.author})</> : null}
+          {props.caption}
+          <span style={{fontStyle: 'italic', fontSize: 'smaller', color: 'gray'}}>
+          {props.author ? <> (Bild: <a target="_blank" referrerPolicy="no-referrer" href={props.source}>{props.author}</a>)</>
```

Comment:
you should only make an `<a>` tag if the image has a source otherwise it will be a dead link.